setup.py

```diff
@@ -14,68 +14,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""simpl packaging and installation."""
+"""simpl packaging, installation, and package attributes."""
 
-import ast
-import re
+import os
 from setuptools import find_packages
 from setuptools import setup
 
 
-DEPENDENCIES = [
+src_dir = os.path.dirname(os.path.realpath(__file__))
+
+about = {}
+with open(os.path.join(src_dir, 'simpl', '__about__.py')) as abt:
+    exec(abt.read(), about)
```

Comment:
I considered changing my comment as it's an over-simplification. What I didn't like about the `ast.literal_eval` solution was the proliferation of regex'es. One of my oft-repeated quotes: solve a problem with regex, now you have two problems. Additionally, the decode -> regex match -> literal_eval muddles things.

Since @samstav was adding even more information, that meant more regexes and the code was already fairly obfuscated to my eyes.

The `exec` approach is crystal clear to me: read the stuff out of this file and stuff it in a dict. That works for me. :)